apps/volunteer/sign_up.py

```diff
@@ -8,13 +8,16 @@
     Markup,
     render_template_string,
 )
+
+from datetime import timedelta
 from flask_login import current_user
 from wtforms import StringField, SelectField, SubmitField, BooleanField
 from wtforms.validators import Required, Email, ValidationError, Optional
 
 from pendulum import parse, period
```

Comment:
as `parse` is no longer being used we can remove it from imports (black should pick this up if you `run_tests` or install the pre-commit hooks)

Reply:
Removed that and fixed my lack of pre-commit. Whoops, my bad :(